packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -279,6 +299,8 @@ export default {
 
     .card-collapse > hr {
         margin-top: 0;
+        background-color: currentcolor;
+        opacity: 0.5;
```

Comment:
a bit confused what does this affect!

Reply:
This is to ensure that the page break also inherits the colour of the text, or else it would always be a dark colour. 
For example with the inherit property:
<img width="768" alt="Screenshot 2024-04-07 at 4 55 42 PM" src="https://github.com/MarkBind/markbind/assets/105090139/94f89710-4edc-4a3b-8079-c9f1ff7ec6df">

without the inherit property:
<img width="768" alt="Screenshot 2024-04-07 at 4 55 21 PM" src="https://github.com/MarkBind/markbind/assets/105090139/4c4a5b49-d3fd-4920-bd95-1a71dcb8d9f7">

As you can see without these changes the page break might sometimes be unnoticeable against the parent's colour